index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Forms</title>
+<link rel="stylesheet" href="styles.css">
+</head>
+<body>
+      <h1 id="title">
+        A Survey Form
+      </h1>
+    <p class="description">
+      Welcome! We hope you have a seamless time here
+    </p>
+   <form id="survey-form">
+       <label id="name-label">First name: </label><br>
+       <input type="text" id="name" required></input>
+       <input type="submit" name="name"></input><br>
+       <label id="email-label">Email</label><br>
+       <input type="email" id="email" name="name" required></input>
+       <input type="submit" name="email"></input><br>
+       <label id="number-label">Phonenumber:</label><br>
+       <input type="number" placeholder="enter your phone number" min="0" max="50" id="number" name="number" required></input>
+       <input type="submit" name="number"></input><br>
+       <select id="dropdown">select
+        <option></option>
+        <option></option>
+     </select>
+     <input type="radio" value="HTML" name="Buttons">Button 1</input>
+     <input type="radio" value="CSS" name="Buttons">Button 2</input>
+
+     <input rows = 5 cols = 10 > Enter msg </input>
+     <button id="submit">Submit</button>
+   </form>
```

Comment:
Great job for using the form tag! It portrays your intention well and makes it understandable